go/protocol/keybase1/block.go

```diff
@@ -113,6 +113,22 @@ func (o PutBlockArg) DeepCopy() PutBlockArg {
 	}
 }
 
+type PutBlockAgainArg struct {
+	Folder   string         `codec:"folder" json:"folder"`
+	Ref      BlockReference `codec:"ref" json:"ref"`
```

Comment:
Yeah I understand why we need it; I'm wondering if there's a way we can refactor to make both APIs use this approach.